src/fileTransformer/fileTransformer.ts

```diff
@@ -55,4 +67,21 @@ const tools: Tools = {
 
     return file.content.replace(pattern, replacement)
   },
+  prismaSchema: {
+    addPreviewFlag(params) {
+      if (previewFeaturesPattern.exec(params.file.content)) {
```

Comment:
We don't want to care what order file transformers run in, at least as much as we reasonably can.